sdk/python/packages/flet-core/src/flet_core/navigation_bar.py

```diff
@@ -1,39 +1,36 @@
 from enum import Enum
-from typing import Any, List, Optional, Union, Dict
+from typing import Any, Callable, Dict, List, Optional, Union
 
 from flet_core.adaptive_control import AdaptiveControl
 from flet_core.border import Border
 from flet_core.buttons import OutlinedBorder
 from flet_core.constrained_control import ConstrainedControl
-from flet_core.control import Control, OptionalNumber
+from flet_core.control import Control
 from flet_core.ref import Ref
 from flet_core.types import (
     AnimationValue,
+    ControlEventFunction,
+    MaterialState,
     OffsetValue,
+    OptionalNumber,
     ResponsiveNumber,
     RotateValue,
     ScaleValue,
-    MaterialState,
 )
 
-try:
-    from typing import Literal
-except ImportError:
-    from typing_extensions import Literal
-
 
 class NavigationBarLabelBehavior(Enum):
-    """Defines how the destinations' labels will be laid out and when they'll be displayed."""
+    "Defines how the destinations' labels will be laid out and when they'll be displayed."
 
     ALWAYS_SHOW = "alwaysShow"
     ALWAYS_HIDE = "alwaysHide"
     ONLY_SHOW_SELECTED = "onlyShowSelected"
 
 
-class NavigationDestination(Control):
+class NavigationBarDestination(Control):
```

Comment:
Thanks for working on this. Let's not forget that many people are using this, so let's follow the "deprecation before removal" principle to avoid breaking working code.
So, we will have both classes for now, and remove the deprecated one in Flet v1.0
**Idea:** leave this `NavigationDestination`  class as it was, create a new class named `NavigationBarDestination` that inherits from `NavigationDestination` and exposes exactly thesame properties.

Reply:
Ha, I just thought about it and was about to do it. I'll create variable named `NavigationDestination` and it's will be deprecated variable

Reply:
@ndonkoHenri Do you know how use deprecated in flet_core? I looked at `utils.py` and didn't understand how use it. 

My code:
```py
NavigationDestination = NavigationBarDestination
```